src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -63,7 +65,11 @@ function ImageRenderer({tnode}: ImageRendererProps) {
     const imagePreviewModalDisabled = htmlAttribs['data-expensify-preview-modal-disabled'] === 'true';
 
     const fileType = FileUtils.getFileType(attachmentSourceAttribute);
-    const fallbackIcon = fileType === CONST.ATTACHMENT_FILE_TYPE.FILE ? Expensicons.Document : Expensicons.Gallery;
+    const fallbackIcon = fileType === CONST.ATTACHMENT_FILE_TYPE.FILE ? Expensicons.Document : Expensicons.GalleryNotFound;
+    const [hasLoadFailed, setHasLoadFailed] = useState(true);
```

Comment:
Is there any specific reason why we are setting `failed` as the default?
```suggestion
    const [hasLoadFailed, setHasLoadFailed] = useState(false);
```

Reply:
@suneox I set the default value of `hasLoadFailed` to true to handle the bug that occurs if the user clicks on the image too quickly.